data/json/monstergroups/bugs.json

```diff
@@ -52,7 +52,7 @@
   {
     "type": "monstergroup",
     "name": "GROUP_BEE",
-    "monsters": [ { "monster": "mon_bee_small", "cost_multiplier": 0 } ]
+    "monsters": [ { "monster": "mon_bee_small", "cost_multiplier": 0 }, { "monster": "mon_bee", "cost_multiplier": 5 } ]
```

Comment:
This is a big balance change spawning advanced mutant bees on day one. Please remove this bit. Otherwise this all looks good to me.

Reply:
Great call, didn't think about that. Will make a separate group for beehive with scary bees.